ai-ml-samples/interactive/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Can you add the required libraries commented out, including the note that Dataproc Serverless includes them by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done</p>